libs/common-const/src/cowprotocolTokenLogoUrl.ts

```diff
@@ -1,6 +1,6 @@
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
-const tokenUrlRoot = 'https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images'
+const tokenUrlRoot = 'https://files.cow.fi/token-lists/images'
```

Comment:
Nitpick:
Would be great to move `https://files.cow.fi` into env vars and use it everywhere from env vars.

p.s. the var can be defined in those files:
<img width="176" height="159" alt="image" src="https://github.com/user-attachments/assets/26c1bb0b-ee93-4e84-8a6f-c05e20a3b176" />